src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2835,6 +2836,31 @@ void statusOfTlrRequestShouldBeChangedIfAssociatedItemCheckedIn() {
     assertThat(requestAssociatedWithFirstItem.getString("status"), is(OPEN_AWAITING_PICKUP));
   }
 
+  @Test
+  void pageRequestShouldNotChangeItemStatusIfFailsWithoutRequestDate() {
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    var item = itemsFixture.basedUponSmallAngryPlanet();
+    var requester = usersFixture.steve();
+
+    Response response = requestsClient.attemptCreate(new RequestBuilder()
+      .page()
+      .fulfilToHoldShelf()
+      .withRequesterId(requester.getId())
+      .withItemId(item.getId())
+      .withPickupServicePointId(pickupServicePointId)
+      .withInstanceId(item.getInstanceId())
+      .itemRequestLevel()
+      .withHoldingsRecordId(item.getHoldingsRecordId()).withRequestDate(null));
```

Comment:
```suggestion
      .withHoldingsRecordId(item.getHoldingsRecordId())
      .withRequestDate(null));
```
let's stick on one style

Reply:
sure, thanks